api/spec/src/billing/customer.tsp

```diff
@@ -0,0 +1,117 @@
+using TypeSpec.Http;
+using TypeSpec.OpenAPI;
+
+namespace OpenMeter.Billing;
+
+@route("/api/v1/billing/customer")
+@tag("Billing (Experimental)")
+interface Customers {
+  /**
+   * List customer overrides
+   */
+  @get
+  @summary("List customer overrides")
+  @operationId("billingListCustomerOverrides")
+  list(
+    @query
+    billingProfile?: Array<ULID>,
+
+    ...OpenMeter.QueryPagination,
+    ...OpenMeter.QueryLimitOffset,
+    ...OpenMeter.QueryOrdering<CustomerOverrideOrderBy>,
+  ): OpenMeter.PaginatedResponse<CustomerOverride> | OpenMeter.CommonErrors;
+
+  /**
+   * Create/update a new customer override.
+   */
+  @post
+  @route("/{customerIdOrKey}")
+  @summary("Create/update a customer override")
+  @operationId("billingCreateCustomerOverride")
+  create(
+    @path
+    customerIdOrKey: ULIDOrKey,
+
+    @body
+    request: CustomerWorkflowOverride,
+  ): {
+    @statusCode _: 200;
+    @body body: CustomerOverride;
+  } | OpenMeter.NotFoundError | OpenMeter.CommonErrors;
+
+  /**
+   * Get a customer override by id.
+   */
+  @get
+  @route("/{customerIdOrKey}")
```

Comment:
Do we need to support the key here? I don't think it will be called in latency-sensitive or UI path where lookup is undesired.